MAVSDK-CSharp/MAVSDK.CSharp.ConsoleClient/Program.cs

```diff
@@ -22,20 +22,22 @@ static async Task Main(string[] args)
              *     - emit an event only when the value changes
              *     - discard the altitudes lower than 0
              */
+            /*
             var telemetryPlugin = new TelemetryPlugin(Host, Port);
```

Comment:
For now, I only populated `templates/call.j2`, so the telemetry plugin doesn't have any implementation. That would be the next step.